.github/history.json

```diff
@@ -2898,7 +2898,7 @@
         },
         {
           "pr": "7622",
-          "title": "[FIX] Error when acessing settings before ready",
```

Comment:
This file is generated automatically, so you should list the PRs to needing the title change and some one with access to them will do, this case the PR is https://github.com/RocketChat/Rocket.Chat/pull/7622

EDIT
I did the changes in all PRs you listed here, they will be available on next release